_posts/2016-03-16-AngularTS.md

```diff
@@ -0,0 +1,413 @@
+---
+layout: post
+author: kevin_van_houtte
+title: 'AngularTS: A new look @ Angular'
+image: /img/angularts.jpg
+tags: [TypeScript, AngularJS]
+category: AngularJS, TypeScript
+comments: true
+---
+
+## Combining the best of two worlds.
+
+Since my introduction to the heroic AngularJS framework at Devoxx around 4 years ago.
+I was intrigued and set for an adventure.
+With Angular 2 release in sight we have to prepare ourselves with the migrating road map coming up.
+One of the core changes in angular, was changing the code language to TypeScript.
+This blog will cover the use of angular components in TypeScript.
+But what is this TypeScript? 
+TypeScript is a superset of JavaScript that focuses on strong typing and the new ES6 features: classes, interfaces and modules.
+Like in common OO languages like Java and C# these features aren’t new.
```

Comment:
Remove the first Like (In common ...) or "Like in common OO languages such as Java ..."